dev/breeze/src/airflow_breeze/utils/github.py

```diff
@@ -54,6 +54,11 @@ def download_file_from_github(tag: str, path: str, output_file: Path, timeout: i
     if not get_dry_run():
         try:
             response = requests.get(url, timeout=timeout)
+            if response.status_code == 403:
+                get_console().print(
+                    f"[error]The {url} is not accessible.This may be caused by network issues or VPN settings"
```

Comment:
nit: We recently got some 403 from Github because we hit their throttle limit. Maybe update the message to also this as possible cause?

Reply:
Its actually pretty useless to print `response.content` - it deviates entirely from the actual error

Reply:
Throttle limit, yes. Good idea